docker-compose/dev/apps/config/sampler.json

```diff
@@ -7,5 +7,23 @@
     "port": 7233,
     "namespace": "pocket-ml-testbench",
     "task_queue": "sampler"
-  }
+  },
+  "timeouts": {
```

Comment:
"timeouts" should be nested inside a "LLM" category, it should not be a base configuration of the sampler app.

Reply:
But the only information accesed is the service. I added a 'type' attr. Now it only take the value of 'llm'. In the future itg can vary.